src/widgets/FridgeDisplayInfo.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { UIDatabase } from '../database';
+import { formatTemperatureExposure, formatTemperature } from '../utilities/formatters';
+
+import { FlexRow } from './FlexRow';
+import { ChevronDownIcon } from './icons';
+import { SimpleLabel } from './SimpleLabel';
+
+import { WHITE, SUSSOL_ORANGE } from '../globalStyles';
+import { vaccineStrings } from '../localization';
+
+export const FridgeDisplayInfo = ({ fridge, isActive }) => {
+  const { description, currentTemperature, temperatureExposure, numberOfBreaches } = fridge;
+  return (
+    <View style={localStyles.container}>
+      <FlexRow flex={1} justifyContent="space-between" alignItems="center">
+        <FlexRow flex={1} justifyContent="space-evenly">
+          <ChevronDownIcon color={isActive ? WHITE : SUSSOL_ORANGE} style={localStyles.icon} />
+          <SimpleLabel
+            label={description}
+            size="large"
+            text={formatTemperature(currentTemperature)}
```

Comment:
Nit: could separate this out and wrap in `useMemo` with `currentTemperature` as a dep?

Reply:
thanks for these - will do a round of performance tweaking if needed. For now I think I'll just hold off. I think it's obvious you disagree but on this one in particular my thoughts are that the difference between an equality check and just calling this method is so minimal it's not worth it? 🤷‍♂ mainly because it's a primitive !